dev_tools/scripts/db_integrity_test.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
+
+# Insert some test data in the db
+docker exec db psql -U postgres -d test -c "INSERT INTO local_ega.main(submission_file_path,submission_user,submission_file_extension,status,encryption_method) VALUES('test.c4gh','dummy','c4gh','INIT','CRYPT4GH');"
+
+# Build, run and execute the pg_basebackup action
+docker container run --rm -i --name pg-backup --network=host $(docker build -f dev_tools/Dockerfile-backup -q -t backup .) /bin/sda-backup --action pg_basebackup
+
+# Find the name of the copy in the S3 and check the length
+DBCOPY=$(s3cmd -c dev_tools/s3conf ls s3://dumps/ | grep ".tar" | cut -d '/' -f4)
```

Comment:
```suggestion
DBCOPY=$(s3cmd -c dev_tools/s3conf ls s3://dumps/ | grep ".enc" | cut -d '/' -f4)
```
It's better to use another suffix than `tar` since it is both compressed and encrypted.